.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
Can you add two more lines on this file? 

```
--color
--format documentation
--require spec_helper

```

Reply:
Done 👍  thanks